libs/db-browser/src/test/java/com/oceanbase/tools/dbbrowser/env/BaseTestEnv.java

```diff
@@ -133,7 +156,7 @@ private static String generate() {
         }
         log.info("hostName={}, removeSpecial={}, HOST_NAME={}", hostName, removeSpecial, hostName);
         long currentMillis = System.currentTimeMillis() % 1000000;
-        return "dbbrowser_" + hostName + "_" + currentMillis;
+        return "db_" + hostName + "_" + currentMillis;
```

Comment:
why should we change this prefix?

Reply:
Because in Oracle mode, the length of the currently generated schema name has exceeded the maximum length supported by Oracle, and an exception will be thrown during creation.